lib/inc/drogon/Cookie.h

```diff
@@ -41,6 +41,31 @@ class DROGON_EXPORT Cookie
     {
     }
 
+    /// Constructor
+    /**
+     * @param key key of the cookie
+     * @param value value of the cookie
+     */
+    Cookie(const std::string &key, std::string &&value)
```

Comment:
For the function having more than one parameters, I perfer to use one value parameters function instead of 2**N overrides.